en-us/helpers/converters.md

```diff
@@ -14,11 +14,67 @@ Commonly used **converters** that allow the data to be modified as it passes thr
 
 | Converter | Purpose |
 | --- | --- |
+|BoolToObjectConverter | Converts a boolean value into an object. The converted value is selected between the values of TrueValue and FalseValue properties |
 |BoolToVisibilityConverter | Converts a boolean value into a Visibility enumeration |
 |CollectionVisibilityConverter | Converts a collection into a Visibility enumeration (Visible if the given collection is not empty or null) |
+|EmptyCollectionToObjectConverter | Converts a collection into an object. The converted value is selected between the values of EmptyValue and NotEmptyValue properties |
+|EmptyStringToObjectConverter | Converts a string into an object. The converted value is selected between the values of EmptyValue and NotEmptyValue properties |
 |StringFormatConverter | Converts a source object to the formatted string version |
 |StringVisibilityConverter | Converts a string value into a Visibility enumeration (if the value is null or empty returns a collapsed value) |
 
+## Example
+`BoolToObjectConverter` can be used to generalize the behavior of `BoolToVisibilityConverter` by allowing to pass the two values it can return.
```

Comment:
I think something has gone awry with your merge as these shouldn't be showing up as new additions.